app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create!(game_params.merge(white_player_id: current_user.id))
+    if @game.save
```

Comment:
`create` and its bang counterpart both generate an object and attempt to save it, so calling `save` here is redundant. Instead, it should suffice to check whether the object was actually saved as part of the `create` call by calling `persisted?`.